express-api/src/services/ches/chesServices.ts

```diff
@@ -158,7 +153,12 @@ const sendEmailAsync = async (email: IEmail, user: SSOUser): Promise<IEmailSentR
       email.delayTS += Number(cfg.ches.secondsToDelay);
     }
   }
-  email.to = email.to.filter((a) => !!a);
+  email.to = cfg.ches.overrideTo
+    ? cfg.ches.overrideTo
+        .split(';')
+        .map((email) => email.trim())
+        .filter((email) => !!email)
+    : [user.email];
```

Comment:
I think this last part is supposed to be email.to.filter((a) => !!a) still. Otherwise we never send things to the to address.